src/mono/mono/mini/mini-arm64.h

```diff
@@ -271,6 +275,8 @@ typedef struct {
 	host_mgreg_t gregs [PARAM_REGS + 1];
 	/* Floating registers */
 	double fregs [FP_PARAM_REGS];
+	/* Context registers */
+	host_mgreg_t cregs [CTX_REGS];
```

Comment:
These can be stored in gregs, same as on amd64.

Reply:
Updated to use gregs.